src/Kconfig

```
@@ -37,6 +37,16 @@ config GOLIOTH_OTA_MAX_NUM_COMPONENTS
     help
         Maximum number of components in an OTA manifest.
 
+config GOLIOTH_FW_OBSERVATION_RETRY_MAX_DELAY_S
+    int "Golioth maximum delay before retrying firmware update observation"
+    default 3600 # 1 hour
```

```suggestion
    default 300 # 5 minutes
```

I tested this at 1 hour and it certainly works. But I think the default might be better at a lower value to that it's more likely a user will see the log message when troubleshooting issue. It is fully configurable when a longer value is needed in a project.

---

I'm fine either way. This is another config option that we'll probably want to provide some guidance around best practices eventually.